requirements.txt

```
@@ -0,0 +1,7 @@
+mkdocs==1.5.3
+mkdocs-git-revision-date-localized-plugin==1.2.2
+mkdocs-glightbox==0.3.6
+mkdocs-material==9.5.4
+mkdocs-material-extensions==1.3.1
+pymdown-extensions==10.7
+Jinja2>=2.11.1
```

@anneschuth a suggestion to "shorten" the requirements list. What do you think of this?
```suggestion
mkdocs-material[git]~=9.5.4
mkdocs-glightbox~=0.3.6
```

---

Not sure it will work, I saw it fail without suffiently specifying versions before (this is the bug I fixed for the Algoritmekader)